.travis.yml

```diff
@@ -51,7 +51,7 @@ matrix:
         - ninja -Cout/linux/x64/release
         - cp ./out/linux/x64/release/escargot ./escargot
         - travis_wait 30 tools/run-tests.py --arch=x86_64 jetstream-only-simple chakracore
-        - tools/run-tests.py --arch=x86_64 jetstream-only-cdjs sunspider-js internal octane jsc-stress v8 spidermonkey regression-tests es2015 intl
+        - travis_wait 30 tools/run-tests.py --arch=x86_64 jetstream-only-cdjs sunspider-js internal octane jsc-stress v8 spidermonkey regression-tests es2015 intl
```

Comment:
Do these tests require a longer running time than before?

Reply:
Oops, I left it here. I think it van be removed.